### PR TITLE
Blind fix for the radare2 pkg

### PR DESCRIPTION
**Pull request by trufae**
The shlr lib is not suposed to be installed in the system, and the list after prepareInstalledDevelLibs was incomplete. i just updated this, is the CI going to verify that im not pushing something wrong? i dont have time for testing or learning how to build ports from haiku right now, so i would appreicate some help/feedback here if needed.

thanks

### CODE REVIEW

**Comment by pulkomandy** [comment]
The CI does not check this far, so you'd have to wait until this is merged and then watch https://build.haiku-os.org/buildmaster/master/x86_64/ to see the result of the build. If the build fails, the previous recipe remains in the package repository.

**Comment by trufae** [comment]
Thank you!

**Comment by pulkomandy** [comment]
Build in progress here: https://build.haiku-os.org/buildmaster/master/x86_64/?buildrunDir=6000&viewMode=expanded

**Comment by korli** [comment]
libs should first be removed in $libDir

**Comment by trufae** [comment]
Oh! Theres a duplicated lib and this makes the build fail

File: dev-util/radare2/radare2-5.4.0.recipe [hunk 1]
```diff
@@ -277,7 +277,7 @@ INSTALL()
 	mv $prefix/data/radare2 $docDir
 	rm -rf $prefix/share
 
-	prepareInstalledDevelLibs libr_shlr libr_anal libr_asm libr_bin \
+	prepareInstalledDevelLibs libr_anal libr_asm libr_bin libr_debug libr_main \
 	libr_bp libr_config libr_cons libr_core libr_crypto libr_debug libr_egg \
```
Comment (Begasus) [inline dev-util/radare2/radare2-5.4.0.recipe:281]: libr_debug is called twice here:

```diff
-	prepareInstalledDevelLibs libr_anal libr_asm libr_bin libr_debug libr_main \
+	prepareInstalledDevelLibs libr_anal libr_asm libr_bin \
```

`libr_main` should be moved further in the line (alphabetically)

Comment (Begasus) [reply dev-util/radare2/radare2-5.4.0.recipe:281]: Should be fixed in https://github.com/haikuports/haikuports/pull/6185

**Pull request merged**